pkg/defaults/defaults.go

```diff
@@ -308,7 +308,7 @@ const (
 	// AnnotateK8sNode is the default value for option.AnnotateK8sNode. It is
 	// enabled by default to annotate kubernetes node and can be disabled using
 	// the provided option.
-	AnnotateK8sNode = true
+	AnnotateK8sNode = false
```

Comment:
> but I'm just unsure what we actually use the annotations for.

that's why I'm only changing the default to `false` I'm also uncertain but from what I could tell we don't use the annotations for anything critical since the information is store in the Cilium Node.